sentry_sdk/integrations/django/views.py

```diff
@@ -0,0 +1,53 @@
+import functools
+
+from sentry_sdk.hub import Hub
+from sentry_sdk._types import MYPY
+
+if MYPY:
+    from typing import Any
+
+    from django.urls.resolvers import ResolverMatch
+
+
+def patch_resolver():
+    # type: () -> None
+    try:
+        from django.urls.resolvers import URLResolver
+    except ImportError:
+        from django.core.urlresolvers import RegexURLResolver as URLResolver
+
+    from sentry_sdk.integrations.django import DjangoIntegration
+
+    old_resolve = URLResolver.resolve
+
+    def resolve(self, path):
+        # type: (URLResolver, Any) -> ResolverMatch
+        hub = Hub.current
+        integration = hub.get_integration(DjangoIntegration)
+
+        if integration is None or not integration.middleware_spans:
+            return old_resolve(self, path)
+
+        with hub.start_span(op="django.urls.resolve"):
```

Comment:
I'm not sure this is of much use. If all the resolver does is call the view, isn't it going to be very similar in duration to the `django.view` span?

Reply:
yeah let's remove it. resolver is probably fast enough, usually